openshift/config/setup-playbook.yaml

```diff
@@ -31,42 +34,57 @@
     fail:
       msg: k8s_config_environment_level must be set to 'dev', 'test', or 'prod'
     when: >-
-      k8s_config_environment_level|default('') not in ['dev','test','prod']
+      k8s_config_environment_level|default('') not in ['dev', 'test', 'prod']
 
-  - name: Get cluster ingress configuration
+  - name: Get cluster version
     k8s_info:
       api_version: config.openshift.io/v1
-      kind: Ingress
-      name: cluster
-    register: _k8s_info_get_cluster_ingress
+      kind: ClusterVersion
+      name: version
+    register: _k8s_info_cluster_version
 
-  - name: Get anarchy-k8s-config default AnarchyRunner
+  - name: Get OpenShift API server information
     k8s_info:
-      api_version: anarchy.gpte.redhat.com/v1
-      kind: AnarchyRunner
-      name: default
-      namespace: anarchy-k8s-config
-    register: _k8s_info_get_anarchy_k8s_config_default_anarchy_runner
+      api_version: config.openshift.io/v1
+      kind: ClusterOperator
+      name: openshift-apiserver
+    register: _k8s_info_cluster_operator_openshift_apiserver
 
-  - name: Get anarchy-operator default AnarchyRunner
+  - name: Get cluster ingress configuration
     k8s_info:
-      api_version: anarchy.gpte.redhat.com/v1
-      kind: AnarchyRunner
-      name: default
-      namespace: anarchy-operator
-    register: _k8s_info_get_anarchy_operator_default_anarchy_runner
+      api_version: config.openshift.io/v1
+      kind: Ingress
+      name: cluster
+    register: _k8s_info_cluster_ingress
 
-  - name: Set k8s_info fact
+  - name: Set k8s_cluster_info with set_k8s_cluster_info and discovered information
     set_fact:
-      k8s_info:
+      k8s_cluster_info:
+        cluster_domain: >-
+          {{ _k8s_info_cluster_ingress.resources[0].spec.domain
+           | regex_replace('^apps\.', '')
+           | regex_replace('cluster-(.*)\.\1', '\1')
```

Comment:
do we want `cluster-`  hardcoded here ? i know we discuss that it's often hardcoded in our code, but it's just an historical convention

Reply:
It is a bit weird... We can remove it, we would just need to make sure we fix the path in babylon-private to match.

Reply:
Changed so it will now preserve the `cluster-GUID` bit when it is there.